src/sentry/services/hybrid_cloud/auth/model.py

```diff
@@ -35,16 +35,17 @@
 
 
 class RpcApiKey(RpcModel):
-    id: int
-    organization_id: int
-    key: str
-    status: int
-    allowed_origins: List[str]
-    label: str
+    id: int = -1
+    organization_id: int = -1
+    key: str = ""
+    status: int = 0
+    allowed_origins: List[str] = Field(default_factory=list)
+    label: str = ""
+    scope_list: List[str] = Field(default_factory=list)
 
 
 class RpcAuthenticatorType(IntEnum):
-    API_KEY_AUTHENTICATION = 0
+    UNUSUED_ONE = 0
```

Comment:
Typo:
```suggestion
    UNUSED_ONE = 0
```

Maybe also prepend an underscore?

Reply:
🤷 to be honest in a very short period, if I get this in, the entire thing disappears.  We won't be performing any authenticators this way.